remoteappmanager/docker/container_manager.py

```diff
@@ -165,7 +158,7 @@ def _start_container(self, user_name, image_name, volumes):
         # build the dictionary of keyword arguments for create_container
         container_name = _generate_container_name("remoteexec",
                                                   user_name,
-                                                  image_name)
+                                                  mapping_id)
```

Comment:
Ah this does restrict the username+mapping_id to be a unique identifier for one container.  But as far as I understand, we allow multiple containers per user+application+policy, no?  This configuration will raise an error when someone wants to create a second instance of the same application+policy.